### PR TITLE
Update App.js - To add missing supported OS

### PR DESCRIPTION
**Pull request by camilamacedo86**
[sc-78497]

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -191,13 +191,13 @@ cat install.sh | sudo bash -s airgap
                       CentOS 7.4, 7.5, 7.6, 7.7, 7.8, 7.9, 8.1, 8.2, 8.3 (CentOS 8.x requires Containerd)
                     </li>
                     <li className="u-fontSize--small u-color--dustyGray u-fontWeight--medium u-lineHeight--normal">
-                      RHEL 7.4, 7.5, 7.6, 7.7, 7.8, 7.9, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6, 8.7, 9.0, 9.1 (RHEL 8.x and 9.x require Containerd)
+                      RHEL 7.4, 7.5, 7.6, 7.7, 7.8, 7.9, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6, 8.7, 8.8, 9.0, 9.1, 9.2 (RHEL 8.x and 9.x require Containerd)
                     </li>
                     <li className="u-fontSize--small u-color--dustyGray u-fontWeight--medium u-lineHeight--normal">
                       Rocky Linux 9.0, 9.1 (Rocky Linux 9.x requires Containerd)
```
Comment (emosbaugh) [inline src/components/App.js:197]: can you add 9.2 here?

**Review by emosbaugh — CHANGES_REQUESTED** [review]

File: src/components/App.js [hunk 2]
```diff
@@ -191,13 +191,13 @@ cat install.sh | sudo bash -s airgap
                       CentOS 7.4, 7.5, 7.6, 7.7, 7.8, 7.9, 8.1, 8.2, 8.3 (CentOS 8.x requires Containerd)
```
Comment (emosbaugh) [inline src/components/App.js:191]: can you add 8.4 here

**Review by emosbaugh — DISMISSED** [review]

File: src/components/App.js (hunk 2 above)
Comment (laverya) [reply src/components/App.js:191]:
```suggestion
                      CentOS 7.4, 7.5, 7.6, 7.7, 7.8, 7.9, 8.1, 8.2, 8.3, 8.4 (CentOS 8.x requires Containerd)
```

File: src/components/App.js [hunk 3]
```diff
@@ -188,16 +188,16 @@ cat install.sh | sudo bash -s airgap
...
                     <li className="u-fontSize--small u-color--dustyGray u-fontWeight--medium u-lineHeight--normal">
-                      CentOS 7.4, 7.5, 7.6, 7.7, 7.8, 7.9, 8.1, 8.2, 8.3 (CentOS 8.x requires Containerd)
+                      CentOS 7.4, 7.5, 7.6, 7.7, 7.8, 7.9, 8.1, 8.2, 8.3, 8.4 (CentOS 8.x requires Containerd)
                     </li>
                     <li className="u-fontSize--small u-color--dustyGray u-fontWeight--medium u-lineHeight--normal">
-                      RHEL 7.4, 7.5, 7.6, 7.7, 7.8, 7.9, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6, 8.7, 9.0, 9.1 (RHEL 8.x and 9.x require Containerd)
+                      RHEL 7.4, 7.5, 7.6, 7.7, 7.8, 7.9, 8.1, 8.2, 8.3, 8.4, 8.5, 8.6, 8.7, 8.8, 9.0, 9.1, 9.2 (RHEL 8.x and 9.x require Containerd)
                     </li>
                     <li className="u-fontSize--small u-color--dustyGray u-fontWeight--medium u-lineHeight--normal">
                       Rocky Linux 9.0, 9.1 (Rocky Linux 9.x requires Containerd)
```
Comment (camilamacedo86) [inline src/components/App.js:197]:
```suggestion
                      Rocky Linux 9.0, 9.1, 9.2 (Rocky Linux 9.x requires Containerd)
```

**Review by laverya — APPROVED** [review]

**Pull request merged**